scripts/rs-init.sh

```diff
@@ -1,6 +1,13 @@
 #!/bin/bash
 
-MONGO_NODES=(mongo1t:27027 mongo2t:27028 mongo3t:27029)
+COUNT=$(env | grep -c "VIRTUAL_ENV")
+if [ $COUNT -eq 0 ]; then
```

Comment:
@HeriLFIU, it's not always safe to assume that if it has $VIRTUAL_ENV then it should use `mongo1t:27027 mongo2t:27028 mongo3t:27029`. 

`MONGO_NODES` here value should be by default `mongo1t:27027 mongo2t:27028 mongo3t:27029`, if the env var is not set, otherwise you can set `MONGO_NODES` value on `mongo-setup` docker compose yml file service definition `MONGO_NODES=mongo1LT:27037 mongo2LT:27038 mongo3LT:27039`. I'm not sure if this shell variable parsing interpolation will work but it's something to also explore and confirm. If parsing these values becomes too hard, then we can also try to set another env var where if it's defined then it'll use `MONGO_NODES=(mongo1LT:27037 mongo2LT:27038 mongo3LT:27039)`